sdk/monitor/azure-monitor-query/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
   - `MetricsBatchQueryClient` has been renamed to `MetricsClient`. ([#33958](https://github.com/Azure/azure-sdk-for-python/pull/33958))
   - Reordered the arguments for the async `MetricsClient` constructor so that `endpoint` is now the first positional argument. ([#33752](https://github.com/Azure/azure-sdk-for-python/pull/33752))
   - Positional arguments in `MetricsClient.query_resources` are now required keyword-only arguments. ([#33958](https://github.com/Azure/azure-sdk-for-python/pull/33958))
+  - The `resource_uris` argument in `MetricsClient.query_resources` has been renamed to `resource_ids`. ([#34760](https://github.com/Azure/azure-sdk-for-python/pull/34760))
```

Comment:
Looks like we can treat it in a non-breaking way?

Reply:
Hmm, how so? If someone using the previous beta used `resource_uris` as a keyword argument, then this rename would cause a breakage (giving the user a TypeError).

Reply:
`resource_uris` was a beta feature not GAed one?

I asked this because I saw the last one in your changelog was 1.2.1.

Reply:
If it was a beta feature, then never mind. :)

Reply:
Ah, yea, `1.2.1` was a patch release based on the `1.2.0` release (needed a security fix). This is just breaking a change against the last `1.3.0` beta release, so no breaking a GA version. :)